obspy/io/rg16/tests/test_read_rg16.py

```diff
@@ -72,10 +72,12 @@ def test_headonly_option(self):
 
     def test_starttime_endtime_option(self):
         """
-        Test the the options starttime and endtime
+        Test the options starttime and endtime
         """
         t1 = UTCDateTime(2017, 8, 9, 16, 0, 15)
         t2 = UTCDateTime(2017, 8, 9, 16, 0, 45)
+        t3 = UTCDateTime(2017, 8, 9, 16, 0, 1)
+        t4 = UTCDateTime(2017, 8, 9, 16, 0, 22)
```

Comment:
So these variables basically are sorted like `t3 < t1 < t4 < t2`. That's incredibly confusing, to me at least, please change